### PR TITLE
Enclose URLs in angle brackets

### PR DESCRIPTION
**Pull request by maurolepore**
> URLs should be enclosed in angle brackets, e.g. ‘https://www.r-project.org’: see also Specifying URLs._
--https://cran.r-project.org/doc/manuals/r-release/R-exts.html

### CODE REVIEW

File: index.Rmd [hunk 1]
```diff
@@ -9,7 +9,7 @@ biblio-style: apalike
 link-citations: yes
 github-repo: ropenscilabs/dev_guide
 url: 'https\://ropensci.github.io/dev_guide/'
-description: "Extended version of the rOpenSci packaging guide."
+description: "Extended version of the rOpenSci packaging guide. This book is a guide for authors, maintainers, reviewers and editors of rOpenSci. The first section of the book contains our guidelines for creating and testing R packages. The second section is dedicated to rOpenSci's software peer review process: what it is, our policies, and specific guides for authors, editors and reviewers throughout the process. The third and last section features our best practice for nurturing your package once it has been onboarded: how to collaborate with other developers, how to document releases, how to promote your package and how to leverage GitHub as a development platform. The third section also features a chapter for anyone wishing to start contributing to rOpenSci packages."
 cover-image: images/cover.png
```
Comment (maelle) [inline index.Rmd]: why does this change appear here?

Comment (maurolepore) [reply index.Rmd]: It's not intended. I'll see what's going on. Sorry.

Comment (maelle) [reply index.Rmd]: np

Comment (maurolepore) [reply index.Rmd]: @maelle, I don't know why that text appears where it does. My commit https://github.com/ropensci/dev_guide/pull/169/commits/20eb0b41a420c424c2fca6ec8d0f72abd2d35aa8 doesn't seem to have done it.

What am I missing here?

Comment (maelle) [reply index.Rmd]: not sure but maybe your dev branch was not up to date? The last contributor name on up-to-date dev is Hugo Gruson https://github.com/ropensci/dev_guide/blob/dev/index.Rmd

Comment (maurolepore) [reply index.Rmd]: > maybe your dev branch was not up to date
You are right! I reseted my branch with upstream/dev, then cherry-picked my commit and forced push. It should now be good.

Comment (maelle) [reply index.Rmd]: Thank you!

**Comment by maelle** [comment]
Thanks!

**Pull request merged**